app/main.py

```diff
@@ -18,6 +18,7 @@
 
 import django_setup  # always keep this first
 
+import fix_path
```

Comment:
Maybe it's simpler to directly insert the content of fix_path.py here? It's only used here (after you remove it from unit_tests.py) and it's just one line.